website/src/content/deployment/runtime-configuration.mdx

```diff
@@ -63,6 +63,20 @@ An example configuration for local development:
 }
 ```
 
+## Using environment variables
+
+Values in an environment configuration JSON file can contain references to environment variables. This can be useful to avoid duplication between various `env.json` files for multiple different environments. References are specified with a special expansion like syntax `${}` while additionally prefixing the environment variable name with `env:`.
```

Comment:
Nit: I feel that this paragraph is still a bit confusing for users that do not know what this is. In my suggestion I basically took what was in the Prisma docs, as it's short and clear. Also the use case is explained below, we don't need to mention it here ("_This can be useful to avoid duplication..._")

Reply:
It depends a bit what the reader or audience is. The Prisma docs is definately a developer with database experience. Here we might have somebody with less experience a a developer just spinning up an demo app for tomorrow. As a result I wanted to spend some more words. 

Feel free to push a commit to adjust. I value the feedback but also made 6-8 commits already for a one paragraph documentation addition which was approved for three others :)